package.json

```diff
@@ -25,7 +25,7 @@
     "ember-browserify": "1.1.11",
     "ember-cli": "~2.5.0",
     "ember-cli-app-version": "^1.0.0",
-    "ember-cli-code-coverage": "0.3.2",
+    "ember-cli-code-coverage": "srowhani/ember-cli-code-coverage",
```

Comment:
The excludes prop wasn't working correctly. Tried some stuff in a fork, and got it to work. On my machine, the coverage works fine, but coveralls can't seem to parse the `lvoc.info` that my fork produces.